scrollingpagerindicator/src/main/java/ru/tinkoff/scrollingpagerindicator/RecyclerViewAttacher.java

```diff
@@ -38,30 +38,53 @@
      * +------------------------------+
      */
     public RecyclerViewAttacher() {
-        currentPageLeftCornerX = 0; // Unused when centered
-        centered = true;
+        this(null);
     }
 
     /**
      * Use this constructor if current page in recycler isn't centered.
      * All pages must have the same width.
      * Like this:
-     *
+     * <p>
      * +-|----------------------------+
      * | +--------+  +--------+  +----|
      * | | current|  |        |  |    |
      * | |  page  |  |        |  |    |
      * | +--------+  +--------+  +----|
      * +-|----------------------------+
-     *   | currentPageLeftCornerX
-     *   |
+     * | currentPageLeftCornerX
+     * |
```

Comment:
Revert comment changes please, it's broken here. And remove additional "<p>"

Reply:
done